AEPMessaging/Sources/MessagingRulesEngine.swift

```diff
@@ -36,19 +37,209 @@ class MessagingRulesEngine {
             return
         }
 
+        // TEMP
+        //        let rules = [testRuleString, testMultipleRuleString]
+
         var messagingRules: [LaunchRule] = []
         for rule in rules {
-
-            guard let processedRule = JSONRulesParser.parse(rule.data(using: .utf8) ?? Data()) else {
+            guard let processedRule = JSONRulesParser.parse(rule.data(using: .utf8) ?? Data(), runtime: runtime) else {
                 continue
             }
 
             // TODO: handle remote assets caching (here or in UIServices?)
-
             messagingRules.append(contentsOf: processedRule)
         }
 
         rulesEngine.replaceRules(with: messagingRules)
         Log.debug(label: MessagingConstants.LOG_TAG, "Successfully loaded \(messagingRules.count) message(s) into the rules engine.")
     }
+
+    let testRuleString = """
```

Comment:
nit: should these test json strings be removed?

Reply:
they're in there for now, i will be using them for a test later and move them to a more appropriate location